src/content/blog/kai-deep-dive-2024/index.mdx

```diff
@@ -0,0 +1,171 @@
+---
+author: jwmatthews
+date: 2024-05-07T07:00:00Z
+description: >-
+  Konveyor AI (Kai) Introduction
+featured: true
+image: header.png
+tags:
+  - Konveyor
+  - Kai
+  - GenAI
+  - AppModernization
+title: Kai - Generative AI Applied to Application Modernization
+---
+
+**Author**: [John Matthews](https://github.com/jwmatthews)
+
+### Overview
+
+Konveyor AI or "Kai", is an early effort of Generative AI applied to Application Modernization being explored under the Konveyor Ecosystem at [konveyor-ecosystem/kai](https://github.com/konveyor-ecosystem/kai).
+
+Kai implements a [Retrieval Augmented Generation (RAG)](https://www.promptingguide.ai/techniques/rag) approach that leverages data from Konveyor to help generate code suggestions to aid migrating legacy code bases to a different technology. The intent of this RAG approach is to shape the code suggestions to be similar to how an organization has solved problems in the past, without additional fine-tuning of the model.
```

Comment:
It might be worth it to include a little one-sentence summary of RAG since it's so relevant to the approach

Reply:
Also it might make sense to explicitly refer to the Large Language Model (LLM) here instead of just referring to it as a "model". Could then link to some doc on that as well